src/kOS.Safe.Test/Serialization/JSONFormatterTest.cs

```diff
@@ -1,105 +1,24 @@
 ﻿using kOS.Safe.Encapsulation;
 using kOS.Safe.Serialization;
 using NUnit.Framework;
+using kOS.Safe.Utilities;
 
 namespace kOS.Safe.Test.Serialization
 {
     [TestFixture]
-    public class JSONFormatterTest
+    public class JSONFormatterTest : FormatterTest
     {
-        [SetUp]
-        public void Setup()
-        {
-
-        }
-
-        [Test]
-        public void CanSerializeLexicons()
-        {
-            var lex = new Lexicon();
-            var nested = new Lexicon();
-
-            lex[new StringValue("key1")] = new StringValue("value1");
-            lex[new StringValue("key2")] = new ScalarIntValue(1);
-            lex[new StringValue("key3")] = nested;
-
-            nested[new StringValue("nested1")] = new StringValue("nested1value");
-            nested[new StringValue("nested2")] = new StringValue("nested2value");
-
-            Lexicon deserialized = Deserialize(Serialize(lex)) as Lexicon;
-
-            Assert.AreEqual(new StringValue("value1"), deserialized[new StringValue("key1")]);
-            Assert.AreEqual(new ScalarIntValue(1), deserialized[new StringValue("key2")]);
-            Assert.IsTrue(deserialized[new StringValue("key3")] is Lexicon);
-            Assert.AreEqual(new StringValue("nested1value"), (deserialized[new StringValue("key3")] as Lexicon)[new StringValue("nested1")]);
+        protected override IFormatReader FormatReader {
+            get {
+                return JsonFormatter.ReaderInstance;
+            }
         }
 
-        [Test]
-        public void CanSerializeLists()
-        {
-            var list = new ListValue();
-            var nested = new ListValue();
-
-            list.Add(new StringValue("item1"));
-            list.Add(new ScalarIntValue(2));
-            list.Add(nested);
-
-            nested.Add(new StringValue("nested1"));
-
-            ListValue deserialized = Deserialize(Serialize(list)) as ListValue;
-
-            Assert.AreEqual(new StringValue("item1"), deserialized[0]);
-            Assert.AreEqual(new ScalarIntValue(2), deserialized[1]);
-            Assert.IsTrue(deserialized[2] is ListValue);
+        protected override IFormatWriter FormatWriter {
+            get {
+                return JsonFormatter.WriterInstance;
+            }
```

Comment:
(See elsewhere in the PR comments for the comment with the word "curly brace" in big letters.)